### PR TITLE
Added bug fix,known issues and improvements to the Release notes

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.10 only
- [Preview](http://file.del.redhat.com/asakthiv/5.1.0-rel-notes/windows_containers/windows-containers-release-notes-5-x.html#wmco-5-1-0)

**NOTE**
The node certificates addition in the Rel Notes , does not require a SME/peer review as it is already done ref: https://github.com/openshift/openshift-docs/pull/46239  

@jfrancoa @jrvaldes @mtnbikenc @selansen ptal

cc @mburke5678

### CODE REVIEW

**Comment by mtnbikenc** [comment]
Would it be worthwhile to add in the `New features and improvements` section that Windows Server 2022 support is being added for vSphere and Bare metal?

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 1]
```diff
@@ -23,13 +23,34 @@ ifndef::openshift-origin[]
...
 
+[id="wmco-5-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 5.1.0
+
+This release of the WMCO is now available with a bug fix and an improvement. The components of the WMCO 5.1.0 were released in link:https://access.redhat.com/errata/RHBA-2022:95425-01[RHBA-2022:95425-01].
+
+=== Bug fix
+
+Previously, the reverse DNS lookup of Windows Bring-Your-Own-Host (BYOH) instances failed when the node's external IP was present without pointer records (PTR). With this release, if the PTR record is not present in the first node IP address,
+the WMCO looks in the other node addresses until a reverse lookup record is found. As a result, the reverse configuration of Windows BYOH instances succeed when node external IP address is present without a PTR record.
```
Comment (mburke5678) [inline windows_containers/windows-containers-release-notes-5-x.adoc:41]: s/succeed/succeeds
s/when node/when the node

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 2]
```diff
@@ -23,13 +23,34 @@ ifndef::openshift-origin[]
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2081825[**BZ#2081825**])
+
+=== Known Issue
+
+Windows machine sets can't scale up when the `publicIP machineset` field is set to `False` on Microsoft Azure. This will be fixed in a future release of WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2091642[**BZ#2091642**])
```
Comment (mburke5678) [inline windows_containers/windows-containers-release-notes-5-x.adoc:46]: s/can't/cannot

Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-5-x.adoc:46]: s/the `publicIP machineset` field is set to `False/the `publicIP` parameter is set to false in the machine set ??
@jfrancoa @jrvaldes @mtnbikenc

**Comment by Amrita42** [comment]
> Would it be worthwhile to add in the `New features and improvements` section that Windows Server 2022 support is being added for vSphere and Bare metal?

Yes good point, adding that.

File: windows_containers/windows-containers-release-notes-5-x.adoc (hunk 2 above)
Comment (mansikulkarni96) [reply windows_containers/windows-containers-release-notes-5-x.adoc]: @Amrita42 change suggested by Michael looks good to me, the wording should be:
the publicIP` parameter is set to false in the machine set

**Review by jrvaldes — COMMENTED** [review]
@Amrita42  thanks for working on this, comments added. PTAL

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 3]
```diff
@@ -23,13 +23,38 @@ ifndef::openshift-origin[]
...
+
+=== Bug fix
+
+Previously, the reverse DNS lookup of Windows Bring-Your-Own-Host (BYOH) instances failed when the node's external IP was present without pointer records (PTR). With this release, if the PTR record is not present in the first node IP address,
+the WMCO looks in the other node addresses until a reverse lookup record is found. As a result, the reverse configuration of Windows BYOH instances succeed when the node external IP address is present without a PTR record.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2081825[**BZ#2081825**])
+
+=== Known Issue
+
+Windows machine sets can't scale up when the `publicIP machineset` is set to `False` on Microsoft Azure. This will be fixed in a future release of WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2091642[**BZ#2091642**])
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply windows_containers/windows-containers-release-notes-5-x.adoc:46]: Thanks @mansikulkarni96

[the remark this answers is not in the document]
Comment (Amrita42) [reply windows_containers/windows-containers-release-notes-5-x.adoc:46]: > `publicIP machineset` may be challenging to read, I suggest mentioning `publicIP` as a field of the machineSet providerSpec.

Yes modified as `the publicIP parameter is set to false in the machineset`  as suggested in previous comment by Mansi

**Comment by Amrita42** [comment]
@mansikulkarni96  @jrvaldes  ptal and if all is good , please add a lgtm label.

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 4]
```diff
@@ -23,13 +23,38 @@ ifndef::openshift-origin[]
...
+
+=== Bug fix
+
+Previously, the reverse DNS lookup of Windows Bring-Your-Own-Host (BYOH) instances failed when the node's external IP was present without pointer records (PTR). With this release, if the PTR record is not present in the first node IP address,
+the WMCO looks in the other node addresses until a reverse lookup record is found. As a result, the reverse configuration of Windows BYOH instances succeed when the node external IP address is present without a PTR record.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2081825[**BZ#2081825**])
+
+=== Known Issue
+
+Windows machine sets can't scale up when the `publicIP` parameter is set to `False` in the machine set. This issue is tracked by (link:https://bugzilla.redhat.com/show_bug.cgi?id=2091642[**BZ#2091642**]).
```
Comment (jrvaldes) [inline windows_containers/windows-containers-release-notes-5-x.adoc:46]: s/machine sets/machineSets

Comment (jrvaldes) [reply windows_containers/windows-containers-release-notes-5-x.adoc:46]: s/`False`/`false`

Comment (jrvaldes) [reply windows_containers/windows-containers-release-notes-5-x.adoc:46]: s/machine set/machineSet

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 5]
```diff
@@ -23,13 +23,38 @@ ifndef::openshift-origin[]
...
 
+[id="wmco-5-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 5.1.0
+
+This release of the WMCO is now available with a bug fix and a few improvement. The components of the WMCO 5.1.0 were released in link:https://access.redhat.com/errata/RHBA-2022:95425-01[RHBA-2022:95425-01].
+
+=== Bug fix
+
+Previously, the reverse DNS lookup of Windows Bring-Your-Own-Host (BYOH) instances failed when the node's external IP was present without pointer records (PTR). With this release, if the PTR record is not present in the first node IP address,
+the WMCO looks in the other node addresses until a reverse lookup record is found. As a result, the reverse configuration of Windows BYOH instances succeed when the node external IP address is present without a PTR record.
```
Comment (EricPonvelle) [inline windows_containers/windows-containers-release-notes-5-x.adoc:41]: Lines 40 and 41 are hard returns. Can you make this all one line?

File: windows_containers/windows-containers-release-notes-5-x.adoc (hunk 4 above)
Comment (EricPonvelle) [inline windows_containers/windows-containers-release-notes-5-x.adoc:46]: s/can't/cannot

**Review by EricPonvelle — CHANGES_REQUESTED** [review]
I had two small items, but you may need to ping the current peer review as I am unsure if release notes can have passive voice. It makes sense that we'd allow it since it's all talking about improvements, but please verify this.

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 6]
```diff
@@ -23,13 +23,38 @@ ifndef::openshift-origin[]
 
 {productwinc} is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
 
-You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
```
Comment (EricPonvelle) [inline windows_containers/windows-containers-release-notes-5-x.adoc:26]: is not supported is passive voice

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 7]
```diff
@@ -23,13 +23,38 @@ ifndef::openshift-origin[]
...
 For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
 
-If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support. 
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-5-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 5.1.0
+
+This release of the WMCO is now available with a bug fix and a few improvement. The components of the WMCO 5.1.0 were released in link:https://access.redhat.com/errata/RHBA-2022:95425-01[RHBA-2022:95425-01].
```
Comment (EricPonvelle) [inline windows_containers/windows-containers-release-notes-5-x.adoc:36]: Passive voice:

is now available
were released

File: windows_containers/windows-containers-release-notes-5-x.adoc (hunk 4 above)
Comment (EricPonvelle) [reply windows_containers/windows-containers-release-notes-5-x.adoc:46]: Passive voice:

is set
is tracked

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 8]
```diff
@@ -23,13 +23,38 @@ ifndef::openshift-origin[]
...
+
+=== New features and improvements
+[id="wmco-5-1.0-node-certificates"]
+==== Windows node certificates are updated
+
+With this release, the WMCO updates the Windows node certificates when the kubelet client certificate authority (CA) certificate is rotated.
+
+==== Windows Server 2022 support
+
+With this release, Windows Server 2022 is now supported on VMware vSphere and Bare metal.
```
Comment (EricPonvelle) [inline windows_containers/windows-containers-release-notes-5-x.adoc:56]: is now supported is passive voice.

Maybe something like "Release XX now supports ..."

**Review by EricPonvelle — COMMENTED** [review]
I highlighted the instances of passive voice.

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Review by EricPonvelle — APPROVED** [review]

**Review by jrvaldes — APPROVED** [review]
LGTM

**Pull request merged**